common/src/main/java/com/wynntils/models/lootrun/type/MissionType.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.lootrun.type;
+
+public enum MissionType {
+    CLEANSING_GREED("Cleansing Greed", 'e'),
+    HIGH_ROLLER("High Roller", 'e'),
+    MATERIALISM("Materialism", 'e'),
+    ORPHIONS_GRACE("Orphion's Grace", '9'),
+    CLEANSING_RITUAL("Cleansing Ritual", '5'),
+    EQUILIBRIUM("Equilibrium", '5'),
+    INNER_PEACE("Inner Peace", '5'),
+    BACKUP_BEAT("Backup Beat", 'a'),
+    STASIS("Stasis", 'a'),
+    GAMBLING_BEAST("Gambling Beast", 'c'),
+    REDEMPTION("Redemption", 'c'),
+    ULTIMATE_SACRIFICE("Ultimate Sacrifice", 'c'),
+    FAILED("Failed", '4'),
+    ;
```

Comment:
```suggestion
    FAILED("Failed", '4');
```

Reply:
I also don't like that "Failed" is a mission type. It's not. I know it can happen, but I think it should be handled specially. This goes for "unknown" as well, since this class hard-codes all missions, it can easily happen that Wynn adds new ones, then, the mod crashes, errors, etc.

Reply:
To be clear, I am not a 100% sure what would be the best here. Maybe, having some state class for mission state with the states, "not started", "failed", "started", "finished", with the mission either being a mission type, or an "unknown mission".

Reply:
In my recent commit, I have aimed to address this. Please let me know if you want me to do this differently.